File: week-02/day-02/reverse.py
# - Create a variable named `aj`
#   with the following content: `[3, 4, 5, 6, 7]`
# - Reverse the order of the elements in `aj`
# - Print the elements of the reversed `aj`

aj = [3, 4, 5, 6, 7]

print(aj)

for i in range(len(aj)//2):
    a = aj[i]
    aj[i] = aj[len(aj) - i - 1]
    aj[len(aj) - i - 1] = a

print("reversed: ")

print(aj)

File: week-07/day-2/reverse.js
'use strict';
// - Create a variable named `aj`
//   with the following content: `[3, 4, 5, 6, 7]`
// - Reverse the order of the elements in `aj`
// 		- do it with the built in method
//		- do it with creating a new temp array and a loop
// - Print the elements of the reversed `aj`

let aj = [3, 4, 5, 6, 7];

console.log(aj.reverse());

function revArr(normal){
  let reversed = [];
  let len = normal.length;
  for(let i = 0; i < len; i++){
    reversed.push(normal.pop());
  }
  return reversed;
}

console.log(revArr(aj));